modules/core/client/controllers/core.client.controller.sidenav.left.js

```diff
@@ -0,0 +1,23 @@
+(function() {
+  'use strict';
+
+  angular
+    .module('core')
+    .controller('SideNavLeftController', SideNavLeftController);
+
+  SideNavLeftController.$inject = ['Authentication', 'menuService', '$mdSidenav'];
```

Comment:
Cannot use Authentication Here otherwise core will depend on Users.  This needs to be a blank placeholder controller for just the demo and the UserSettingsController should be injected into the view in the users.run() function.